addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -117,8 +117,8 @@ def _l10n_it_edi_check_ordinary_invoice_configuration(self, invoice):
             errors.append(_("Tax representative partner %s of %s must have a tax number.", seller.l10n_it_tax_representative_partner_id.display_name, seller.display_name))
 
         # <1.4.1>
-        if not buyer.vat and not buyer.l10n_it_codice_fiscale and buyer.country_id.code == 'IT':
-            errors.append(_("The buyer, %s, or his company must have a VAT number and/or a tax code (Codice Fiscale).", buyer.display_name))
+        if not buyer.vat and not (buyer.l10n_it_codice_fiscale and buyer.country_id.code == 'IT'):
+            errors.append(_("The buyer, %s, or his company must have a VAT number and/or an Italian tax code (Codice Fiscale).", buyer.display_name))
```

Comment:
@lordkrandel Not sure why this change is here.  This only says that if the buyer is from Italy you should at least have a codice fiscale or a vat and you make it like as a foreigner you also need a vat.

Reply:
Removed.